src/plugins/auto/plugin_config.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "utils/plugin_config.hpp"
+
+namespace MultiDevicePlugin {
+const std::set<std::string> PluginConfig::_availableDevices = {"AUTO", "CPU", "GPU", "TEMPLATE", "NVIDIA", "VPUX", "MULTI", "HETERO", "mock"};
```

Comment:
@yangwang201911 is this list correct for now?

Reply:
I think so.